tests/integration/test_integration.py

```diff
@@ -133,17 +126,17 @@ async def test_given_no_config_when_post_deployment_tasks_and_authorise_charm_th
         Returns:
             None
         """
-        await self.deploy_charm(ops_test, charm)
         vault_unit = ops_test.model.units["vault-k8s/0"]
         await ops_test.model.wait_for_idle(apps=[APPLICATION_NAME], status="blocked", timeout=1000)
```

Comment:
I don't think we need this check. Previous test ensured this.

Reply:
True, removed.